src/components/RouteWrapper.tsx

```diff
@@ -273,6 +274,13 @@ export const ROUTES: RouteProperties[] = [
         description: "Arjun can find query parameters for URL endpoints.",
         category: "Web Application Testing",
     },
+    {
+        name: "Arpscan",
+        path: "/tools/arpscan",
+        element: <Arpscan />,
+        description: "Scan the local network for ARP entries",
```

Comment:
Requires a full stop at the end of the sentence. 

- [ ] Add full stop.

Reply:
Fixed it.